package.json

```diff
@@ -29,6 +29,7 @@
     "url": "https://github.com/inrupt/lit-pod.git"
   },
   "devDependencies": {
+    "@inrupt/vocab-common-rdfext": "^0.4.9",
```

Comment:
I'm not a big fan of using `rdfext` in the module name, since the `NamedNode` class is defined in the RDFJS interface that both rdflib and rdfext support. I'd suggest `"@inrupt/vocab-common-lite`